pysat/tests/instrument_test_class.py

```diff
@@ -118,14 +118,6 @@ class InstTestClass():
     """Provides standardized tests for pysat instrument libraries.
     """
 
-    def setup(self):
-        """Runs before every method to create a clean testing setup."""
-        self.package = pysat.instruments
-
-    def teardown(self):
-        """Runs after every method to clean up previous testing."""
-        del self.package
-
```

Comment:
These lines aren't needed for the base class.  Any test that's actually run has its own setup and teardown.  These are not run according to coveralls.